build-system/npm-publish/write-package-files.js

```diff
@@ -197,6 +197,18 @@ async function writeReactJs() {
   }
 }
 
+/**
+ * todo(kvchari): temporarily copy styles to root of each package to support importing styles from root
+ * Remove when we begin properly proxying style imports.
+ * See issue from more information:
+ */
+function copyCssToRoot() {
+  const extDir = path.join('extension', extension, '1.0');
+  const preactCssDist = path.join(extDir, 'dist', 'styles.css');
+  const preactCssRoot = path.join(extDir, 'styles.css');
+  copyFileSync(preactCssDist, preactCssRoot);
```

Comment:
Maybe make `copyCssToRoot()` an `async` function and replace this line with an `await copyFile(...)`?

Reply:
done